cloudsim-plus/src/main/java/org/cloudsimplus/vmtemplates/package-info 2.java

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
Execute a git pull to get the changes I made directly from your PR and then a git fetch and git merge to get updates from the official repository. 
After that, delete this file. It's just duplicated, and push the changes again.